scripts/aptrepo-create.bash

```diff
@@ -17,7 +17,7 @@ else
     export ARCHS=${2}
 fi
 if [[ ${#} -lt 3 ]]; then
-    export DISTROS=(trusty)
+    export DISTROS=(xenial)
```

Comment:
I would recommend we remove the defaults entirely and just update the README to have people put in the correct distros. Alternatively, we should add a message here so we don't get a silent parameter substitution that people aren't aware of.